src/app/components/group-types-panel/group-types-panel.component.html

```diff
@@ -1,8 +1,12 @@
 <!-- future iteration -->
 <div class="filter" *ngFor="let item of groupTypes">
-  <div class="first-level" (click)='setGroupType(item)' id='{{item.group}}'>
-    <h3>
-      {{ item.name }}
-    </h3>
-  </div>
+  <a [routerLink]="[]" [queryParams]="{typegroup: item.name}">
+    <div class="first-level" (click)='setGroupType(item)'
+      id="{{item.group}}"
+      [class.selected]="item.level == selectedgroupType?.level">
```

Comment:
Any specific reason of not using `[routerLinkActive]="'selected'"` in place of this.